File: C/Iniciante/1018 - Cédulas..c
/*
* Autor: Pierre Vieira
* Data de submissão: 22/08/2018 21:04:11
*/
#include <stdio.h>
int main(){
	int valor;
	int n100, n50, n20, n10, n5, n2, n1, resto;
	scanf("%d", &valor);
	resto = valor;
	n100 = resto/100;
	resto = resto - (n100*100);
	n50 = resto/50;
	resto = resto - (n50*50);
	n20 = resto/20;
	resto = resto - (n20*20);
	n10 = resto/10;
	resto = resto - (n10*10);
	n5 = resto/5;
	resto = resto - (n5*5);
	n2 = resto/2;
	resto = resto - (n2*2);
	n1 = resto/1;
	printf ("%d\n%d nota(s) de R$ 100,00\n%d nota(s) de R$ 50,00\n%d nota(s) de R$ 20,00\n", valor, n100, n50, n20);
	printf("%d nota(s) de R$ 10,00\n%d nota(s) de R$ 5,00\n%d nota(s) de R$ 2,00\n%d nota(s) de R$ 1,00\n", n10, n5, n2, n1);
}

File: C#/Iniciante/1018 - Cédulas.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 14:06:35
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int n, q100, q50, q20, q10, q5, q2, q1;
            n = int.Parse(Console.ReadLine());
            Console.WriteLine(n);
            q100 = n / 100;
            n -= 100 * q100;
            q50 = n / 50;
            n -= 50 * q50;
            q20 = n / 20;
            n -= 20 * q20;
            q10 = n / 10;
            n -= 10 * q10;
            q5 = n / 5;
            n -= 5 * q5;
            q2 = n / 2;
            n -= 2 * q2;
            q1 = n;
            Console.WriteLine($"{q100} nota(s) de R$ 100,00");
            Console.WriteLine($"{q50} nota(s) de R$ 50,00");
            Console.WriteLine($"{q20} nota(s) de R$ 20,00");
            Console.WriteLine($"{q10} nota(s) de R$ 10,00");
            Console.WriteLine($"{q5} nota(s) de R$ 5,00");
            Console.WriteLine($"{q2} nota(s) de R$ 2,00");
            Console.WriteLine($"{q1} nota(s) de R$ 1,00");
        }
    }
}
